integration_tests/e2e/induction/inductionSecurityScenarios.cy.ts

```diff
@@ -18,20 +18,6 @@ context('Security tests for creating and updating Inductions', () => {
   })
 
   describe('Creating Inductions', () => {
-    it('should redirect to auth-error page given user does not have any authorities', () => {
```

Comment:
Has this test started failing? which is why you're removed it? 🤔 
It shouldn't be failing, and is still a valid test - if you are a user with no authorities (you are effectively a read-only user), you should not be able to get to the `/create-induction/....` or `/induction/....` pages

I'm hoping both of these tests still pass and we can keep them

Reply:
Ah, I now understand why you deleted this test, and I agree its the right thing to have done 👍